Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:20.11.1-slim AS build

```

Comment:
Known vulnerabilities
![image](https://github.com/user-attachments/assets/8bd47c1d-128b-40ba-a43c-cc2188977d69)

Reply:
https://hub.docker.com/layers/library/node/20.11.1-slim/images/sha256-2b3042f699adef6d192f2cdaf62eb1621d36792c2ab9f268b0a07745063bafad?context=explore

Reply:
Good catch copy-paste thing. I have upgraded to latest slime version 2.18.0